src/themes/default/elements/icon.overrides

```diff
@@ -36,11 +36,9 @@ i.icon.linkedin.in:before { content: "\f0e1"; }
 i.icon.zoom.in:before { content: "\f00e"; }
```

Comment:
Why was this file edited?

Reply:
I think the issue is I branched off master rather than next